### PR TITLE
feature: added authentication token

### PR DESCRIPTION
**Pull request by Nerocodes**
Ready for review for this [issue](https://github.com/viniciusarre/quote-a-lang-server/issues/5)

### CODE REVIEW

File: src/modules/auth/helper.js [hunk 1]
```diff
@@ -47,3 +53,24 @@ export const signUp = (user, email, password) => {
...
+ * @desc Validate authentication token
+ * @param {object} request
+ * @param {object} response
+ * @param {*} next
+ * @returns {*} json or next
+ */
+export const verifyToken = (req, res) => {
+  const token = req.headers.authorization;
+  if (!token) {
+    res.status(400).json({ error: 'No token provided' });
```
Comment (GeovanniG) [inline src/modules/auth/helper.js]: I believe you may want to exit the function here, i.e., return res.status....

Comment (Nerocodes) [reply src/modules/auth/helper.js]: Thank you for the feedback, it's been updated now.

**Comment by viniciusarre** [comment]
@Nerocodes 
Sorry, I have merged another PR which has generated conflicts... could you fix them? I'm reviewing your solution and we'll probably merge it soon!

**Comment by viniciusarre** [comment]
@Nerocodes 
Great work! I will be merging this for now!
Quick feedback: it still needs to implement the function in order for it to work, as well as returning the token after the user logs in (this usually gets stored in the browser or client, in order to keep the session active)... But other than that, nice work! You've laid the needed groundwork for JWT! 

I will open an issue to follow up on this one soon, let me know if you'd like to take part (I could assign you directly to it ;) )

Thank you again for your contribution!

**Comment by viniciusarre** [comment]
closes #7

**Pull request merged**